learn/documentation.adoc

```diff
@@ -30,6 +30,8 @@
 
 '''
 
+////
+
```

Comment:
If this seems like duplicate content, we can replace it by a conditional (asciidoc supports this) so it only renders
if !(site.pom.latest.version == site.pom.latestFinal.version)
and otherwise it renders something like

== Non Final Releases

Beta1 coming soon.


See also the trick in:
https://github.com/droolsjbpm/optaplanner-website/blob/master/_partials/latestReleases.html.haml#L11